components/picker/stories/template.js

```diff
@@ -136,13 +138,10 @@ export const Template = ({
 				isDisabled,
 				isReadOnly,
 				customClasses,
-				customStyles: {
```

Comment:
Where did these go? 😛

Reply:
@rubencarvalho I removed the inline styling of the Picker because it was displaying the label and button inline and overriding the flex-box styling and disrupting the text wrap especially when the label was displayed left. So I "turned it off" by removing this block. I can bring it back if needed

Reply:
Oh, thanks for the context! I thought it was a forgotten edit 😄